aws/aws-explanation/eks-snaps.rst

```diff
@@ -0,0 +1,26 @@
+Snap Usage in EKS Worker Images
+===============================
+
+The EKS worker images bundle several snaps that provide
+kubernetes components. This page will provide a brief
+explanation on what snaps are bundled and why.
+
+Snaps
+=====
+
+.. list-table::
+   :header-rows: 1
+
+   * - **Snap Name**
+     - **Provides**
+     - **Bundled Since**
+   * - kubelet-eks
+     - kubelet binary - primary configuration mechanism
+     - EKS 1.17
+   * - kubectl-eks
+     - kubectl binary
+     - EKS 1.17
+   * - `ecr-credential-provider <https://github.com/kubernetes/cloud-provider-aws/tree/master?tab=readme-ov-file#aws-credential-provider>`_
```

Comment:
Since the link calls it **AWS** credential provider and not ECR credential provider, it is a little confusing. How about moving the link to the 'Provides' column and using the name as is?